packaging/mariner/azure-nvme-utils.spec.tmpl

```diff
@@ -1,11 +1,11 @@
 Name:           azure-nvme-utils
-Version:        0.1.1
-Release:        1%{?dist}
+Version:        __VERSION__
+Release:        __RELEASE__%{?dist}
```

Comment:
If you don't want to mess with the template you can achieve the same thing with something like:

```suggestion
Version:        %{__git_version}
Release:        %{__git_release}%{?dist}
```
And then alter your build call to
```
rpmbuild -ba --define "__git_version $version" --define "__git_release $release" azure-nvme-utils.spec
```